src/test/scala/fi/oph/koski/virta/VirtaXMLConverterSpec.scala

```diff
@@ -42,6 +42,7 @@ class VirtaXMLConverterSpec extends AnyFreeSpec with TestEnvironment with Matche
       <virta:Osuus>1.000000</virta:Osuus>
     </virta:Koulutusala>
     <virta:Opinnaytetyo>0</virta:Opinnaytetyo>
+    <virta:Luokittelu>1</virta:Luokittelu>
```

Comment:
Onko luokittelu sellainen tieto, jota jokaisessa pyynnössä palautuu XML-datassa? Tuota tapausta varten kannattaa tarkistaa, että onko näin ja sitten kirjoitella sille testcase.

Reply:
Ah joo testi tälle pitikin lisätä. En tiedä onko aina mukana, siks lista on mallissa wrapatty `Option`:iin